Intake/Onboarding/InterestingModules.swift

```diff
@@ -16,24 +16,21 @@ struct InterestingModules: View {
     
     var body: some View {
         SequentialOnboardingView(
-            title: "INTERESTING_MODULES_TITLE",
-            subtitle: "INTERESTING_MODULES_SUBTITLE",
+            title: "The Intake Process",
+            subtitle: "Intake works in 3 simple steps...",
             content: [
                 SequentialOnboardingView.Content(
-                    title: "INTERESTING_MODULES_AREA1_TITLE",
-                    description: "INTERESTING_MODULES_AREA1_DESCRIPTION"
+                    title: "Log in to Firebase",
+                    description: "Sign in or make an account so we can keep your information secure and accessible."
                 ),
                 SequentialOnboardingView.Content(
-                    title: "INTERESTING_MODULES_AREA2_TITLE",
-                    description: "INTERESTING_MODULES_AREA2_DESCRIPTION"
+                    title: "Explain your Chief Complaint",
+                    // Want to add, but makes me fail pull request tests: What brings you in today?
```

Comment:
I'm not exactly sure what you mean by that? Swiftlint just checks if you conform to syntactical best practices in Swift, not the content of a `SequentialOnboardingView`